test/verify/check-machines

```diff
@@ -393,6 +393,48 @@ class TestMachines(MachineCase):
         b.wait_in_text("#vm-subVmTest1-disks-hda-target", "hda")
         b.wait_in_text("#vm-subVmTest1-disks-hdb-target", "hdb")
 
+    def testNetworks(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-networks") # wait for the tab
+        b.click("#vm-subVmTest1-networks") # open the "Networks" subtab
+
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "network")
+        b.wait_in_text("#vm-subVmTest1-network-1-source", "default")
+        b.wait_in_text("#vm-subVmTest1-network-1-target", "vnet0")
+
+        b.wait_in_text("#vm-subVmTest1-network-1-state span", "up")
+
+        # Test add network
+        m.execute("virsh attach-interface --domain subVmTest1 --type network --source default --model virtio --mac 52:54:00:4b:73:5f --config --live")
+        
+        b.wait_present("#vm-subVmTest1-network-2-type")
+        b.wait_in_text("#vm-subVmTest1-network-2-type", "network")
+        b.wait_in_text("#vm-subVmTest1-network-2-source", "default")
+        b.wait_in_text("#vm-subVmTest1-network-2-target", "vnet1")
+        b.wait_in_text("#vm-subVmTest1-network-2-model", "virtio")
+        b.wait_in_text("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:5f")
+
+        b.wait_in_text("#vm-subVmTest1-network-2-state span", "up")
+
+        # Test remove network
+        m.execute("virsh detach-interface --domain subVmTest1 --type network --mac 52:54:00:4b:73:5f --config")
+        print "Restarting vm-subVmTest1, might take a while"
+        b.click("#vm-subVmTest1-reboot-caret")
+        b.wait_visible("#vm-subVmTest1-forceReboot")
+        b.click("#vm-subVmTest1-forceReboot")
```

Comment:
Is there any expectation on result after this step?

Reply:
no